kythe/cxx/indexer/cxx/IndexerASTHooks.cc

```diff
@@ -2354,16 +2368,21 @@ bool IndexerASTVisitor::TraverseBinaryOperator(clang::BinaryOperator* BO) {
   if (auto rhs = BO->getRHS(), lhs = BO->getLHS();
       lhs != nullptr && rhs != nullptr) {
     if (!WalkUpFromBinaryOperator(BO)) return false;
-    auto* lvhead = UsedAsWrite(FindLValueHead(lhs));
+    auto lvhead = UsedAsWrite(FindLValueHead(lhs));
```

Comment:
please use structured bindings here to give relevant names to the returned struct/pair members